ai/ai21labs/v0/config/tasks.json

```diff
@@ -0,0 +1,1056 @@
+            "template"
+          ],
+          "title": "Prompt",
+          "type": "string"
+        },
+        "prompt-images": {
+          "description": "The prompt images (Note: Not supported by AI21labs Models)",
+          "instillAcceptFormats": [
+            "array:image/*"
+          ],
+          "instillUIOrder": 3,
+          "instillUpstreamTypes": [
+            "reference"
+          ],
+          "items": {
+            "type": "string"
+          },
+          "title": "Prompt Images",
+          "type": "array"
+        },
+        "seed": {
+          "description": "The seed (Note: Not supported by AI21labs Models)",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 4,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Seed",
+          "type": "integer"
+        },
+        "system-message": {
+          "default": "You are a helpful assistant.",
+          "description": "The system message helps set the behavior of the assistant. For example, you can modify the personality of the assistant or provide specific instructions about how it should behave throughout the conversation. By default, the model\u2019s behavior is set using a generic message as \"You are a helpful assistant.\"",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillShortDescription": "The system message helps set the behavior of the assistant",
+          "instillUIMultiline": true,
+          "instillUIOrder": 2,
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "title": "System message",
+          "type": "string"
+        },
+        "temperature": {
+          "default": 0.7,
+          "description": "The temperature for sampling",
+          "instillAcceptFormats": [
+            "number"
+          ],
+          "instillUIOrder": 5,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Temperature",
+          "type": "number"
+        },
+        "top-k": {
+          "default": 10,
+          "description": "Top k for sampling (Note: Not supported by AI21labs Models)",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUIOrder": 5,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Top K",
+          "type": "integer"
+        },
+        "top-p": {
+          "default": 0.7,
+          "description": "Top p for sampling",
+          "instillAcceptFormats": [
+            "number"
+          ],
+          "instillUIOrder": 5,
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "title": "Top P",
+          "type": "number"
+        },
+        "n": {
```

Comment:
important
It will influence the response count of text. right?
So, the text should be array.

Reply:
I will remove this field, since there are other vendors support this parameter, but I always set them to 1 (reference: [groq](https://github.com/instill-ai/component/blob/main/ai/groq/v0/tasks.go)).
User should to run multiple times of VDP to generate different output.